modules/rosa-sdpolicy-instance-types.adoc

```diff
@@ -18,10 +18,10 @@ endif::rosa-with-hcp[]
 ifndef::rosa-with-hcp[]
 Single availability zone clusters require a minimum of 3 control plane nodes, 2 infrastructure nodes, and 2 worker nodes deployed to a single availability zone.
 
-Multiple availability zone clusters require a minimum of 3 control plane nodes, 3 infrastructure nodes, and 3 worker nodes. Additional nodes must be purchased in multiples of three to maintain proper node distribution.
+Multiple availability zone clusters require a minimum of 3 control plane nodes, 3 infrastructure nodes, and 3 worker nodes.
 
-
-Control plane and infrastructure nodes are deployed and managed by Red{nbsp}Hat. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss. There are at least 3 control plane nodes that handle etcd- and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes. See the Red{nbsp}Hat Operator support section below for more information about Red{nbsp}Hat workloads that must be deployed on worker nodes.
+Control plane and infrastructure nodes are deployed and managed by Red{nbsp}Hat; these nodes are automatically resized based on the resource utilization. To proactively resize these nodes in anticipation of change in cluster utilization, open a support case. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss. Workloads must be deployed on the worker nodes deployed in the cluster using {product-title} machine pools. You may only run node-level workloads (link:https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/([daemonsets]) on the control plane and infrastructure nodes. You will be responsible for such workloads deployed on these nodes in ensuring they are secure, performant, and compatible with OpenShift versions so that the Service Level Agreement (SLA) for API server availability is not affected. Red{nbsp}Hat may notify you and resize the control plane or infrastructure nodes if the OpenShift components are impacted.
```

Comment:
SME and QE reviews will be needed as some of this information pertains to impactful customer concerns.